tests/runtest.py

```diff
@@ -1029,6 +1098,9 @@ def has_compiler(compiler):
     compilers = []
     if arg.python:
         compilers.append('python')
+    elif arg.rust:
+        if has_compiler('rustc') and os.system('rustup default nightly > /dev/null') == 0:
```

Comment:
Are there nice nightly checking? This because Rust compiler flags `+nightly -Z instrument-mcount` needed.  :)